workers/webpack.js

```diff
@@ -97,6 +98,10 @@ function run(next) {
       .once('end', () => done(null, files))
       .on('file', function found(root, file, cb) {
         //
+        // Run the filter function and if it returns false, dont use that file
+        //
+        if (!factory.filter(file)) return void cb();
+        //
         // Ignore minified files that were found in the directory
         //
         if (file.name.indexOf('.min.') !== -1) return void cb();
```

Comment:
Let's not have two different types of filtering:

- Change `opts.filter` to `opts.filters`
- Define `.filter` prototype function that calls all filters
- Make `file.name.indexOf` a default filter.